tests/services/auth.service.test.js

```diff
@@ -11,15 +11,6 @@ describe('Auth Service', () => {
         authService = new AuthService()
     })
 
-    describe('Interface', () => {
```

Comment:
This is essentially a test of line 11 (`authService = new AuthService()`) - generally we try to avoid testing constructors because it adds little value